code/src/java/pcgen/core/Globals.java

```diff
@@ -681,35 +681,20 @@ else if (baseSize > finalSize)
 	/**
 	 * Return true if resizing the equipment will have any "noticable" effect
 	 * checks for cost modification, armor bonus, weight, capacity
-	 * @param aPC
-	 *
 	 * @param aEq
 	 * @param typeList
 	 * @return TRUE or FALSE
 	 */
-	public static boolean canResizeHaveEffect(final PlayerCharacter aPC, final Equipment aEq, List<String> typeList)
+	public static boolean canResizeHaveEffect(final Equipment aEq, List<String> typeList)
 	{
 		// cycle through typeList and see if it matches one in the BONUS:ITEMCOST|TYPE=etc on sizeadjustment
 		if (typeList == null)
 		{
 			typeList = aEq.typeList();
 		}
-		final List<String> upperTypeList = new ArrayList<>(typeList.size());
-		for (final String type : typeList)
-		{
-			upperTypeList.add(type.toUpperCase());
-		}
 
 		final List<String> resizeTypeList = SettingsHandler.getGame().getResizableTypeList();
-		for (final String rType : resizeTypeList)
-		{
-			if (upperTypeList.contains(rType.toUpperCase()))
-			{
-				return true;
-			}
-		}
-
-		return false;
+		return typeList.parallelStream().map(String::toUpperCase).anyMatch(resizeTypeList::contains);
```

Comment:
I would prefer if you didn't using the streaming api for this, or at the very least don't parallelize it. Parallelization isn't useful in situations where the list size is small, in fact it is often slower